<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDelimitedColumnExtractor</class>
 <widget class="QWidget" name="CDelimitedColumnExtractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delimited Column Extractor</string>
  </property>
  <widget class="QLabel" name="m_pLabelColumns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Columns delimited by this text:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditColumnsText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelExtractColumn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Extract column # </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditExtractColumns">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearAllOutputText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSelAllOutputText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnExtractColumn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXTRACT COLUMN</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxMonoFont">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Monospaced font. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSelectedFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnAsveAs">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
